site/content/agent-app-widgets/js-api.md

```diff
@@ -101,7 +101,191 @@ It appends given message at the end of current conversation input window or into
 LiveChat.putMessage("Hello! This message comes from the App Widget. Press enter to send it!");
 ```
 
+## Add new content to Visitor Details
```

Comment:
Should be this called `Visitor Details`? 'Cause we're using `Customer Details` all over the place... Dunno, just asking.❓

Reply:
Changed to `Customer Details` for now, I think that's a better choice given that we use it elsewhere already.